### ISSUE TITLE
Squash migrations

### ISSUE THREAD

**Issue by bfirsh** [embassynetwork/modernomad#316]
This isn't high priority, but there are a lot of migrations so perhaps we could squash them: https://docs.djangoproject.com/en/2.1/topics/migrations/#migration-squashing

We just need to make sure everyone has migrated recently and then it'll have no impact.

It'll make setting up new environments quicker, and means we can run migrations in tests if we want. (I disabled that a while ago because it was taking a long time!)

**Comment by bfirsh** [comment]
I had a quick go at this and it is non-trivial. Perhaps adding the `elidable` option to add the `RunPython` operations might help.

**Comment by bfirsh** [comment]
It flat out doesn't work.

Getting lots of errors like `ValueError: Related model 'core.Booking' cannot be resolved` because the squashed migration is in the wrong order and points at old names of things etc etc.

**Comment by jessykate** [comment]
gahh that sucks. thanks for taking a spin. i'm happy to have a go too at some point.

**Comment by bfirsh** [comment]
Sweet okay. I don't think it's high priority... the tests don't use migrations so they're hardly ever run from scratch.

**Comment by jonathan-s** [comment]
Yeah, squashing migrations can be a pain if you have lots of migrations overlapping each other. So the way to resolve this would be to reset all migrations at the same time. 

First thing first, shipping to production and getting the rest of the migrations up to reality, which there is a PR for already 🍡
